src/ReflectionClosure.php

```diff
@@ -306,6 +307,17 @@ public function getCode()
                             }
                             $code .= $token[0];
                             break;
+                        case '"':
+                        case '\'':
```

Comment:
I don't think that the single quote case should be added because it cannot contain interpolations.
What about heredoc? A test should be probably added.

Reply:
Right, I will work for this